src/components/Toast/Toast.module.css

```
@@ -1,170 +1,57 @@
-@import '../../design-tokens/mixins.css';
-
+@import '../../design-tokens/mixins.css'
```

```suggestion
@import '../../design-tokens/mixins.css';
```

? not sure if it makes a difference

---

Holy crap it was the semicolon, thanks